testsupport/wait/host.go

```diff
@@ -1064,6 +1064,30 @@ func (a *HostAwaitility) WaitForNotifications(username, notificationType string,
 	return notifications, err
 }
 
+// WaitForNotificationWithName waits until there is an expected Notifications available for the provided user and with the notification type and which match the conditions (if provided).
+func (a *HostAwaitility) WaitForNotificationWithName(notificationName, notificationType string, criteria ...NotificationWaitCriterion) ([]toolchainv1alpha1.Notification, error) {
+	a.T.Logf("waiting for notification with name '%s'", notificationName)
+	var notifications []toolchainv1alpha1.Notification
```

Comment:
Since we're using Get we should only expect a single notification so no need for the slice

Reply:
done in [f571b95](https://github.com/codeready-toolchain/toolchain-e2e/pull/560/commits/f571b955373036242fc903480344d27435a26df1)